doc/internals/contributing.rst

```diff
@@ -105,6 +105,10 @@ These are the basic steps needed to start developing on Sphinx.
        . ~/.venv/bin/activate
        pip install -e .
 
+#. install `pre-commit hooks <https://pre-commit.com/>`__::
```

Comment:
it's an anonymous target to avoid raising warnings during the build, any reason you want me to use a reusable target ?

Reply:
Oh sorry, that's right. Alternatively, you can put the link afterwards (like it is done for the rest of the document):

`` rst
blabla `pre-commit`__::

blablabla

.. __: https://pre-commit.com
```

(personally, I would leave it inline because the line is short enough).